packages/ui/src/components/button.tsx

```diff
@@ -25,8 +25,8 @@ const buttonVariants = cva(
         custom: ''
       },
       size: {
-        default: 'h-8 px-6 py-2',
-        sm: 'h-7 px-3 py-0 text-sm font-normal',
+        default: 'h-8 px-6',
+        sm: 'h-7 px-3 text-sm font-normal',
```

Comment:
@abhinavrastogi-harness I only removed the vertical padding since it doesn’t make sense, as the button’s height is already defined using `h-`
<img width="471" alt="image" src="https://github.com/user-attachments/assets/f6a86d48-bfb2-465b-966d-0664f1cca151">